app/components/energy_tariff_form_title_component/energy_tariff_form_title_component.html.erb

```diff
@@ -0,0 +1,60 @@
+<div class="row">
+  <div class="col-md-12">
+    <%= page_title %>
+  </div>
+</div>
+
+<% if notice? %>
+  <div class="row">
+    <div class="col-md-12">
+      <%= notice %>
+    </div>
+  </div>
+<% end %>
+
+<div class="row">
+  <div class="col energy_tariff">
+    <h4><%= t('schools.user_tariffs.title_partial.heading') %></h4>
+    <table class="table table-sm table-charges">
+      <tbody>
+        <% unless skip_field?(:name) %>
```

Comment:
just wondering if here (and below) could be changed to e.g.
```
<% if show_field?(:name) %>
  ...
<% end %>
```

with the component method doing the inverse of, and replacing, `skip_field?` e.g.
```
  def show_field?(field)
    @skip_fields.exclude?(field)
  end
```